src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DataGridViewColumnTypeEditor.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing.Design;
+
+namespace System.Windows.Forms.Design;
+internal class DataGridViewColumnTypeEditor : UITypeEditor
+{
+    public DataGridViewColumnTypeEditor() : base() { }
+
+    private DataGridViewColumnTypePicker? _columnTypePicker;
+
+    public override bool IsDropDownResizable
+    {
+        get
+        {
+            return true;
+        }
+    }
+
+    public override object EditValue(ITypeDescriptorContext? context, IServiceProvider provider, object? value)
+    {
+        if (provider is not null)
```

Comment:
```suggestion
        if (provider is null)
        {
            return value!;
        }
```

Please review is nullability is defined correctly for EditValue return type. THis could be done in a follow up PR.